tools/PTE/pte-main.js

```diff
@@ -1984,7 +1984,7 @@ async function performance_main() {
                             } else {
                                 output["Test Result"] = "FAIL"
                                 logger.info('[performance_main] Test Output:', JSON.stringify(output, null, 4));
-                                throw new Error('[performance_main] Test ran, but failed with errors. Exiting...');
+                                throw new Error('[performance_main] Test ran, but failed with errors. Exiting... \n pteReport:', JSON.stringify(output, null, 4));
```

Comment:
Will this print both strings? Check the job result logs. I think Error takes only one string, so we need to format the string first (including the json string).

Other examples in the nodejs code:

1.  I think I have seen that this actually fails to format the orgName as %s as expected, and so should probably be fixed too. And there are probably others in this code too. It might make sense to review all the code and reformat them correctly with one update (separately from this).:

  throw new Error('[chaincodeInstall] Failed to send install Proposal in (%s) or receive valid response. Response null or status is not 200. exiting...', orgName);

2.  This works ok because logger formats the orgName correctly as %s, and also joins the formatted string with the err :

      logger.error('[chaincodeInstall] Failed to install chaincode in (%s) due to error: ', orgName, err);

Reply:
will add %s

Reply:
Not sure that would work. Test it. Check the suppported API for Error(); I think you may need to format the string first and pass a single arg to Error().

Reply:
tested and added the changes @scottz64